GettingStarted.md

```diff
@@ -68,10 +73,22 @@ If you've never cloned a repo before then you should consider using the GitHub d
 >
 > `Assets -> Export Package…`
 
-# 3. Adding the MixedRealityToolkit-Unity package in your project
+# 4. Adding the MixedRealityToolkit-Unity package in your project
+
+## 4.1 Launch Unity Editor
 
 Open or create your project in Unity.
 
+## 4.2 Install Unity's Universal Windows Platform integration
+
+`File -> Build Settings` select `Universal Windows Platform`
+
+If `Open Download Page` appears then save exe and install. Otherwise, dismiss modal window.
+
+![Unity Build Settings for UWP](/External/ReadMeImages/BuildSettingsUWP.png)
+
+## 4.3 Import MRTK asset
```

Comment:
Unity's menu is Import Package, let's use that term for clarity

Reply:
fixed